js/actions/AppActions.js

```diff
@@ -1,5 +1,9 @@
-import { MESSAGE_ADD } from '../constants/AppConstants';
+import { MESSAGE_ADD, MESSAGE_CLEAR } from '../constants/AppConstants';
```

Comment:
I'd lean towards the plural (MESSAGES_CLEAR) for this. Can be done in a follow up, though